exchange/wrappers/bitx.js

```diff
@@ -0,0 +1,348 @@
+const BitX = require("bitx");
+const _ = require('lodash');
+const moment = require('moment');
+const log = require('../../core/log');
```

Comment:
can you not import anything from outside the exchange folder? That way people can use Gekko Broker as a standalone lib. Thanks!

Reply:
Alright... i'll push an update later